CONTRIBUTING.md

```diff
@@ -132,8 +137,37 @@ meantime.
 
 It can be helpful to go through older bug reports and make sure that they are
 still valid. Load up an older issue, double check that it's still true, and
-leave a comment letting us know if it is or is not. The [least recently updated sort][lru] is good for finding issues like this.
+leave a comment letting us know if it is or is not. The [least recently
+updated sort][lru] is good for finding issues like this.
+
+Contributors with sufficient permissions on the Rust repo can help by adding
+labels to triage issues:
+
+* Yellow, **A**-prefixed labels primarily relate to what part of the project
+  an issue is **about**. 
+
+* Magenta, **B**-prefixed labels identify bugs which **belong** elsewhere. 
+
+* Green, **E**-prefixed labels flag **easy** or introductory bugs. 
+
+* Red, **I**-prefixed labels warn of serious **issues** with the language
+  itself, such as regressions and slowdowns.
```

Comment:
We have a special I- issue called `I-nominated` which specifically goes through our nomination process where it's discussed at a triage meeting and assigned a priority label. May be something to call out specifically here?

You may also want to reword "the language itself" to something like "the language and libraries"